en/docs/administer/logging-and-monitoring/monitoring/monitoring-server-health.md

```diff
@@ -1,118 +1,163 @@
 
+     ```git clone https://github.com/[git-username]/carbon-health-check```  
+   
+3. Forked Repository is referred to as `<HEALTH_CHECK_HOME>` in this document. Navigate to `<HEALTH_CHECK_HOME>` and build the product.
+    
+      ```mvn clean install```
+      
+4.  Rename `org.wso2.carbon.healthcheck.api.core-<version-number>-SNAPSHOT.jar` found in the `<HEALTH_CHECK_HOME>/components/org.wso2.carbon.healthcheck.api.core/target` directory as `org.wso2.carbon.healthcheck.api.core-<version-number>.jar`  and paste it in the `<APIM_HOME>/repository/components/dropins` directory.
 
-This section guides you through deploying the Carbon Health Check components in a WSO2 Carbon product that does not support this feature by default.
+5.  Copy the webapp `api#health-check#v1.0.war` found in the `<HEALTH_CHECK_HOME>//components/org.wso2.carbon.healthcheck.api.endpoint/target/` directory and paste it in the the `<APIM_HOME>/repository/deployment/server/webapps` directory.
 
-1.  Download the [org.wso2.carbon.healthcheck.server.feature-&lt;version-number&gt;.zip]({{base_path}}/assets/attachments/126562954/126562955.zip) and extract it. This folder is reffered to as `<API_HOME>` in this document.
-2.  Copy the `org.wso2.carbon.healthcheck.api.core-<version-number>.jar` found in the `<API_HOME>/plugins` directory and paste it in the `<PRODUCT_HOME>/repository/components/dropins` directory.
-3.  Copy the webapp `api#health-check#v1.0.war` found in the `<API_HOME>/features/org.wso2.carbon.healthcheck.server_1.0.0` directory and paste it in the the `<PRODUCT_HOME>/repository/deployment/server/webapps` directory.
-4.  **(Optional step)** Copy the `health-check.config.xml` configuration file found in the `<API_HOME>/features/org.wso2.carbon.healthcheck.server_1.0.0` directory to your `<PRODUCT_HOME>/repository/conf/` directory.
+### Configuring and invoking the API
 
-### Configuring the API
+This feature is disabled by default. To enable the API open the `<API-M_HOME>/repository/conf/deployment.toml` configuration file and update it as follows:.
+    
+```
+[carbon_health_check]
+enable = true
+```
 
-This feature is disabled by default. To enable the API, set the `<Enable>` property in the `health-check-config.xml` file to **true** .
+This is an open API that should ideally be blocked at the load balancer level. To invoke it, start the WSO2 product and send a GET request to the health check API. A sample cURL command is shown below.
 
-!!! tip
-If the feature has not been enabled successfully , a request to the API will only return a 200 OK response.
+``` java
+    curl -k -v https://{hostname}:{port}/api/health-check/v1.0/health
+```  
+   
+Data sources health checker and User store health checker will get enabled when you enable health checkers using above global configuration. 
+The properties configured under each health checker will be available for each health checker at runtime.
```

Comment:
what do you mean by the properties will be available at runtime?

Reply:
When the restAPI call is given it will fetch these config values and execute the request on top of these config values.